saleor/checkout/views/summary.py

```diff
@@ -43,6 +42,14 @@ def _handle_order_placement(request, checkout):
             request, pgettext("Checkout warning", "Please review your checkout.")
         )
         return redirect("checkout:summary")
+    except TaxError as tax_error:
+        messages.warning(
+            request,
+            pgettext(
+                "Checkout warning", "Unable to calculate taxes - %s" % str(tax_error)
+            ),
+        )
+        return redirect("checkout:summary")
```

Comment:
We probably should tell the user that something is wrong. Please retry later or contact the site administrator. Otherwise they will just be redirected, retry and never succeed and will not know what's going on. And the store owner will not know as well if they don't check the logs.

Reply:
`messages.warning` will popup warning info with error_msg to the user. Some errors like: "InvalidPostalCode", "InvalidAddress", "MissingAddress" contain important information for user and user will see them (warning notification). Customer will see default msg for the rest of possible errors - `"We are not able to calculate taxes for your order. Please try later"`

Reply:
Is it enough?

Reply:
If the user made it this far into the process and some information is missing I think that's a programming bug and there's little that the user can do to fix it.

Reply:
It is already logged as a warning before `raise TaxError`